src/json.c

```
@@ -0,0 +1,417 @@
+_Static_assert(sizeof #x>1U,"");             \
+_Static_assert(sizeof #x-1U <=               \
+        sizeof ((struct json_kwd *)0)->str,"")
+
+json_define_keyword(false);
+json_define_keyword(null);
+json_define_keyword(true);
+
+#undef json_define_keyword
+
+static struct json_ret
+json_parse_value (struct json_arg arg)
```

Cppcheck (2.10-2)

[style] The function 'json_parse_value' is never used.

(unusedFunction)